pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -109,12 +109,9 @@ func newCmdConfigSetAuthInfo(out io.Writer, options *createAuthInfoOptions) *cob
 				return
 			}
 
-			err := options.run()
-			if err != nil {
-				fmt.Fprintf(out, "%v\n", err)
-			} else {
+			cmdutil.CheckErrOrRun(options.run(), func() {
```

Comment:
Yeah, I don't see what this construct adds over regular Go syntax.  Would prefer just CheckErr